src/model/channel/message.rs

```diff
@@ -1115,9 +1152,12 @@ impl From<&Message> for MessageReference {
     }
 }
 
+#[allow(useless_deprecated)]
```

Comment:
Remove the deprecated if it is useless. Maybe try applying it to `from`?

If nothing works here, don't worry about it, just put `// TODO(next): Remove this`.